frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-import-container/common-import-container.component.html

```diff
@@ -1,14 +1,14 @@
 <div
-  class="input-container"
+  class="import-container"
   (dragover)="onDragOver($event)"
   (dragleave)="onDragLeave($event)"
   (drop)="onDrop($event)"
   [ngClass]="{ 'drag-over': (isDragOver | async) }"
 >
-    <mat-icon>file_upload</mat-icon>
-    <p class="ft-14-600 drag-text">Drag & drop or</p>
-    <button mat-flat-button color="primary" (click)="fileInput.click()">
+    <span class="material-symbols-outlined">upload</span>
```

Comment:
how are you able to get the upload icon here without mat-icon @zackcl just curious.